src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -421,7 +421,7 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
 
                     if (sd is CoreModel.ComponentInfoSubstate componentInfo)
                     {
-                        referencedEntities.MarkSeenGlobalAddress(componentInfo.PackageAddress);
+                        referencedEntities.MarkSeenGlobalAddress((GlobalAddress)componentInfo.PackageAddress);
```

Comment:
I guess `ComponentInfoSubstate` should simply implement `CoreModel.IGlobalAddressPointer`, don't you think?

Reply:
make sense, wasn't aware of that one. pushed commit with these changes.